gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -1606,13 +1662,15 @@ gst_tensor_transform_set_caps (GstBaseTransform * trans,
   /**
    * @todo support 64bit integer and remove the flag orc_supported
    */
-  if (in_config.info.type == _NNS_INT64 ||
-      in_config.info.type == _NNS_UINT64 ||
-      out_config.info.type == _NNS_INT64 ||
-      out_config.info.type == _NNS_UINT64) {
-    filter->orc_supported = FALSE;
-  } else {
-    filter->orc_supported = TRUE;
+  for (i = 0; i < in_config.info.num_tensors; i++) {
+    if (in_config.info.info[i].type == _NNS_INT64 ||
```

Comment:
TODO (later work)
we can control orc acceleration for multi-tensors with in/out type.
I will try to handle each buffer with orc after this PR.